docs/registry.md

```diff
@@ -20,3 +21,23 @@ The endpoint is a list that can contain multiple image registry URLs split by co
 from a registry, containerd will try these endpoint URLs one by one, and use the first working one.
 
 After modify the config file, you need restart the `containerd` service.
+
+## Configure Registry Credentials
+
+`cri` plugin also supports docker like registry credential config.
+
+To configure a credential for a specific registry endpoint, create/modify the
+`/etc/containerd/config.toml` as follows:
+```toml
+[plugins.cri.registry.auths]
+  [plugins.cri.registry.auths."https://gcr.io"]
+    username = ""
+    password = ""
+    auth = ""
+    identitytoken = ""
+```
+The meaning of each field is the same with the corresponding field in `.docker/config.json`.
+
+Please note that auth config passed by CRI takes precedence. The registry
+credential in the config file will only be used when auth config is not specified
+by CRI.
```

Comment:
/s/by CRI/by kubernetes via CRI/
Follow this [link](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) to read about configuring kubernetes pod to use images from private registries.

Reply:
Done.